packages/core/lib/commands/develop/run.js

```diff
@@ -27,10 +27,20 @@ module.exports = async options => {
 
   const config = Config.detect(options);
   const customConfig = config.networks.develop || {};
+  let mnemonic = "";
```

Comment:
I would avoid assigning default values that aren't meaningful.  If you just want to declare the variable and you're going to give it a value later, you can just write `let mnemonic;`.  By instead writing `let mnemonic = "";`, you're implying that the value `""` is meaningful and will be used at some point, which is misleading.